contributing.adoc

```diff
@@ -90,6 +90,12 @@ After a successful build, if you're connected to a Docker daemon, you can build
 make images
 ```
 
+The above command produces a `camel-k` image with name `docker.io/apache/camel-k`.  Sometimes you might need to produce camel-k images that need to be pushed to the custom repository e.g. `docker.io/myrepo/camel-k`, to do that you can pass a parameter `imgDestination` to the make as shown below:
+
+```
+make imgDestination='docker.io/myrepo' images
```

Comment:
would prefer to have the variable name uppercase

Reply:
but traditionally the make variables are passed as camelCased ones; anyways will have it updated

Reply:
so I leave it as it is?

Reply:
yes for the moment, can you fix the conflicts ?

Reply:
fixed them. dont know how it seeped in IIRC i merged them ;)